OMCompiler/Makefile.common

```diff
@@ -46,7 +46,6 @@ $(builddir_inc)/c/fmi1/ \
 $(builddir_inc)/c/fmi2/ \
 $(builddir_inc)/scripting-API \
 $(builddir_java) \
-$(builddir_share)/omc/scripts/OpenTurns/ \
```

Comment:
In case you are confused by the build failures, the issue might be this:

I think the directory `share/omc/scripting/` was being created indirectly due to this line. The line is gone now so the directory is missing when we try to copy other things into it.

There are a couple of ways to fix it, but I think the sanest one is to just change this to `$(builddir_share)/omc/scripts/` instead of removing it completely.

Reply:
ok, thanks